.github/workflows/pytest.yml

```diff
@@ -0,0 +1,42 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
We should be working in the develop branch

Reply:
Noted. We likely still want CI on the master branch at all, so we've specified both. 
```yml
on:
  push:
    branches: [ master, develop ]
  pull_request:
    branches: [ master, develop ]
```